docs/packages/xstate-cli/index.md

```diff
@@ -0,0 +1,23 @@
+# @xstate/cli
+
+The [@xstate/cli package](https://github.com/statelyai/xstate-vscode/tree/master/apps/cli) contains commands for running typegen. It's pretty small right now, but we're planning on adding many cool features.
+
+## Installation
+
+- Run `npm install @xstate/cli`
+
+## Commands
+
+### `xstate typegen <files>`
+
+`xstate typegen "src/**/*.tsx?"`
```

Comment:
Curious: are the quotation marks necessary?

Reply:
Yes, see the commment below